src/test/java/co/com/perficient/project3/service/impl/TeamServiceImplTest.java

```diff
@@ -81,4 +82,12 @@ void delete() {
         teamService.delete(UUID.randomUUID());
         verify(teamRepository, times(1)).deleteById(any(UUID.class));
     }
+
+    @Test
+    void findByName() {
+        when(teamRepository.findOne(any(Predicate.class))).thenReturn(Optional.of(Team.builder().name(NAME).build()));
+
+        Optional<Team> optionalTeam = teamService.findByName(NAME);
+        Assertions.assertThat(optionalTeam).isNotNull();
```

Comment:
https://www.javacodex.com/JUnit/Test-Object-Is-Not-Null

assertNotNull is more convenient than isNotNull()

It is not a alert in sonarLint? just curious